sky/cli.py

```diff
@@ -4359,6 +4364,99 @@ def _delete_benchmark(benchmark: str) -> None:
         progress.refresh()
 
 
+@cli.group(cls=_NaturalOrderGroup, hidden=True)
+def local():
+    """SkyPilot local tools CLI."""
+    pass
+
+
+@local.command('up', cls=_DocumentedCodeCommand)
+@usage_lib.entrypoint
+def local_up():
```

Comment:
Thanks for this suggestion - I considered it, but I think maybe lets keep it in `cli.py` for now because it is experimental and adding it to core.py would expose it to python API users. I can leave a TODO to move to core.py when it is no longer experimental. Alternatively, if you think we should have it in core.py in this PR, I can move it there too.

Reply:
Sounds good! I think having a TODO for moving is good enough. : )